src/components/dialog/content/signin/ApiKeyForm.vue

```diff
@@ -88,14 +88,23 @@ import Message from 'primevue/message'
 import { computed } from 'vue'
 import { useI18n } from 'vue-i18n'
 
-import { COMFY_PLATFORM_BASE_URL } from '@/config/comfyApi'
+import { getComfyPlatformBaseUrl } from '@/config/comfyApi'
+import { remoteConfig } from '@/platform/remoteConfig/remoteConfig'
 import { apiKeySchema } from '@/schemas/signInSchema'
 import { useApiKeyAuthStore } from '@/stores/apiKeyAuthStore'
 import { useFirebaseAuthStore } from '@/stores/firebaseAuthStore'
 
 const authStore = useFirebaseAuthStore()
 const apiKeyStore = useApiKeyAuthStore()
 const loading = computed(() => authStore.loading)
+const comfyPlatformBaseUrl = computed(() => {
+  const runtimeUrl = remoteConfig.value.comfy_platform_base_url
+  if (runtimeUrl && runtimeUrl.length > 0) {
+    return runtimeUrl
+  }
```

Comment:
nit (v1):
Empty strings are already falsy
```suggestion
  if (runtimeUrl) {
    return runtimeUrl
  }
```